SEO: improve description links (#94)
pspdev/pspdev.github.io: index.md

# Getting started
{: .fs-6 .fw-700 }

Install our open source [PlayStation Portable Software Development Kit](installation.html).
{: .fs-5 }

> PSP SDK was already installed? Dive in to [get started your PSP SDK Homebrew development](how_to_use.html).

Reviewer: ```suggestion
Already have the PSP SDK installed? Dive in and [get started with PSP SDK Homebrew development](how_to_use.html).
```

{: .fs-5 }

> Need some inspiration? Check out some our [PSP SDK examples](basic_programs.html).

Reviewer: ```suggestion
Need some inspiration? Check out our [PSP SDK examples](basic_programs.html).
```

{: .fs-5 }

> Is your code is not working? [Debug it using the PSPLink](debugging.html).

Reviewer: ```suggestion
Is your code is not working? [Debug it using PSPLINK](debugging.html).
```

{: .fs-5 }

Want to improve your PSP SDK homebrew development experience? Get some [tips and tricks here](tips_tricks.html).
{: .fs-5 }

# Contact Us
{: .fs-6 .fw-700 }

If you have ideas, suggestions or questions, please don't hesitate to use our [PSPDEV GitHub Discussions](https://github.com/pspdev/pspdev/discussions).
{: .fs-5 }

If you need help or would like to contribute, don't hesitate to join us on [Discord](https://discord.gg/bePrj9W) or open an issue on [GitHub](https://github.com/pspdev/pspdev/issues). See you there!
{: .fs-5 }

> # Made with PSP SDK from PSPDEV community <3

Reviewer: ```suggestion
# Made with PSP SDK by the PSPDEV community <3
```
